include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -968,7 +1101,7 @@ namespace VectorTools
       mass_matrix.vmult_add(rhs, inhomogeneities);
 
       //now invert the matrix
-      ReductionControl     control(rhs.size(), 0., 1e-12, false, false);
+      ReductionControl     control(5.*rhs.size(), 0., 1e-12, false, false);
```

Comment:
What was the reason again for this change? CG should surely converge in `rhs.size()` iterations.